### ISSUE TITLE
API docs for Deferred state that cancellation is new in 10.0, when it's new in 10.1

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4614]
|<img alt="wulczer's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| wulczer reported|
|-|-|
|Trac ID|trac#4614|
|Type|defect|
|Created|2010-07-24 18:01:56Z|

1 character patch attached

Attachments:

* [deferred.apidoc.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/767/767cac3519f103995a6103795daba047714de742/7c590b4e0db53ff0eaa03c508f1f6ef7c34714da.patch) (579 bytes) - added by wulczer on 2010-07-24 18:02:10Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__4614 4614
type__defect defect
reporter__wulczer wulczer
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1279994516000000 1279994516000000
changetime__1279995673000000 1279995673000000
version__None None
owner__ 
cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="wulczer's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|wulczer removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_2"></a>|@jesstess set owner to @jesstess|
|-|-|

Thanks for catching this. I'll apply it.

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_3"></a>|@jesstess set status to `closed`|
|-|-|

(In [29892]) Apply deferred.apidoc.patch

Author: wulczer
Reviewer: jesstess
Fixes: [#4614](https://github.com/twisted/twisted/issues/4614)

Say that deferred cancellation was introduced in 10.1 in the Deferred docstring.

Not 10.0 as originally documented.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
